pkg/users/account-create-dialog.js

```diff
@@ -278,28 +374,48 @@
         const props = {
             id: "accounts-create-dialog",
             title: _("Create new account"),
-            body: <AccountCreateBody state={state} errors={errors} change={change} />
         };
 
+        if (state.dialogLoading) {
+            props.body = (
+                <Bullseye>
+                    <Spinner isSVG />
+                </Bullseye>
+            );
+        } else {
+            props.body = <AccountCreateBody state={state} errors={errors} change={change} shells={shells} />;
+        }
+
         const footer = {
             actions: [
                 {
                     caption: _("Create"),
                     style: "primary",
                     clicked: () => {
-                        return passwd_check(false, state.real_name, state.user_name, state.password, state.password_confirm, state.locked, state.change_passw_force);
+                        return passwd_check(false, false, state.real_name, state.user_name, state.password, state.password_confirm, state.locked, state.home_dir, state.change_passw_force);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230202-125519-871b841d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)